tmt/cli.py

```diff
@@ -102,9 +103,9 @@ def list_commands(self, context: Context) -> List[str]:  # type: ignore[override
     # ignore[override]: expected, we want to use more specific `Context`
     # type than the one declared in superclass.
     def get_command(  # type: ignore[override]
-            self,
-            context: Context,
-            cmd_name: str
+        self,
+        context: Context,
+        cmd_name: str
```

Comment:
Probably unrelated, besides the whitespace change I can't spot the actual change. Same goes for the empty line above.

Reply:
That probably sneaked in just because of rebase with main (it was needed for rpm builds)